### PR TITLE
Auto layout

### PR DESCRIPTION
**Pull request by SteveAndreou**
This pull request addresses issues outlined in #12 

- Added auto layout as a query option.
- Updated documentation to reflect changes to the options.

### CODE REVIEW

**Comment by negue** [comment]
oh awesome! let me check that (need to add github publish for each PR then its easier to test)

File: src/TaskList.svelte [hunk 1]
```diff
@@ -76,6 +77,13 @@ export let scrollingDuration = 2000;
     padding-bottom: 0;
   }
 
+  .nes-container.auto-layout {
+    height: 100%;
+    max-height: 100vh;
+    width: fit-content;
```
Comment (negue) [inline src/TaskList.svelte]: I think the width can stay as it was (full width) only change the height settings 

then it'll only auto size in the height

**Review by negue — CHANGES_REQUESTED** [review]
only the width but then this is LGTM 👍

**Comment by SteveAndreou** [comment]
I've removed the width adjustment so that it will only scale vertically now.

**Comment by negue** [comment]
LGTM 👍

**Pull request merged**